### ISSUE TITLE
Sublime Theme?

### ISSUE THREAD

**Issue by djstein** [arcticicestudio/nord#19]
I saw one for Atom is there any proposal for a Sublime theme?

**Comment by arcticicestudio** [comment]
There are plans for a official port project, but it may take some time.
I've only created some base files for it yet, but Sublime themes are huge (Syntax & UI) and unfortunately my time is limited at the moment to work on larger port projects due to my real life job. There are many features and languages to implement and I want to make sure to create a good quality theme to support as many features as possible including configuration options.

I've added it to the port project epic issue #18 in the *Editors & IDEs* section.
I'll notify you as soon as the specific backlog issue is created so you can track the progress or contribute ideas for this port project 😉

**Comment by scottcorgan** [comment]
This is my "good enough" solution for sublime text. Works well for JS, HTML, and CSS. Keeps it simple with minimal color usage.

https://github.com/scottcorgan/noridsh

**Comment by fjlaubscher** [comment]
Did this a few days ago. Check it out @djstein 
https://github.com/fjlaubscher/nord-sublime-text

**Comment by pspeter3** [comment]
Is there anything that can done to help here since Sublime Text 3 just came out

**Comment by arcticicestudio** [comment]
Currently I'm too busy to spent much time on my projects. I hope to find some time during my one week holidays in october so I'll try to get a minimal setup done.

**Comment by pspeter3** [comment]
No worries, thanks for maintaining this!

**Comment by lee-hyunkyu** [comment]
Made a working [Sublime Text 3 theme](https://github.com/lee-hyunkyu/SublimeNord). It's not perfect but it's good enough.

**Comment by ghost** [comment]
@lee-hyunkyu 

You beat me to it :) 

Just created one myself but I used material theme as my base instead, here is how mine looks.

<img width="1440" alt="screen shot 2017-10-05 at 12 48 40" src="https://user-images.githubusercontent.com/6619631/31225808-b2ed9a66-a9cb-11e7-8c54-00ef582af666.png">

**Comment by lee-hyunkyu** [comment]
@carlevison  

Wow, gotta say that yours does look better. Any chance that you’d make your theme public?

**Comment by ghost** [comment]
@lee-hyunkyu 

I don't see why not :) here you go

https://github.com/carlevison/Nord-Sublime-Text

The only thing I haven't done is changed the colour of the tabs as I don't use them, but I feel you should be fully capable of managing that yourself.

If you search for Roboto Mono in Material-Theme.sublime-theme I commented it out as it's not everyones cup of tea, if you like it maybe you could change it to your preferred font of choice and font size.

It isn't perfect but will do until @arcticicestudio releases his official version :)

**Comment by lee-hyunkyu** [comment]
@carlevison 

Thanks!

**Comment by ghost** [comment]
@lee-hyunkyu 

I didn't make it clear but I tweaked the Nord colour scheme quite a bit, and to make it easier I renamed it to Material-Theme.tmTheme to make it easier to activate using Material Theme: Activate theme.

**Comment by arcticicestudio** [comment]
I've finally got some free time to start working on this :smile: 

The color scheme (`.tmTheme`) is almost done, only tweaking the support for the [Babel][babel] package to get as close as possible to the awesome JetBrains WebStorm highlighting.

~~The initial version of the port project will "*only*" include the color scheme, but I think it should work/match with the colors of some UI themes like [Spacegray][spaceg] or Material [Darker][dark] / [Palenight][pale] in the meantime I try to complete a kind of Nord UI theme concept and understand this (creepy) UI theme system.~~

**Edit**: Alright, I've decided to support both the UI- and color scheme for the initial version now. The UI theme system is really verbose and don't understand their intent to implement it this way, but I'm making ~~good~~moderate progress ~~so this shouldn't take too long~~. Also the new added default [Adaptive][adaptive] UI theme adapts to the currently active color scheme.

I'll try to port the UI design of the [Nord Atom UI][naui] as far as possible and maybe include some code from the awesome Material theme.

Regarding the UI theme/color scheme preferences:
I'm aware that it is possible to implement preferences and I'd like to add a feature like in arcticicestudio/nord-atom-syntax#47 , but I need some time to read their dev docs to get to know the plugin API.

The repository should be available tomorrow at the latest.

[babel]: https://packagecontrol.io/packages/Babel
[spaceg]: http://kkga.github.io/spacegray/
[dark]: http://equinsuocha.io/material-theme/#/darker
[pale]: http://equinsuocha.io/material-theme/#/darker
[naui]: https://atom.io/themes/nord-atom-ui
[adaptive]: https://www.sublimetext.com/blog/articles/the-adaptive-theme

**Comment by arcticicestudio** [comment]
The [repository][repo] is available. You can track arcticicestudio/nord-sublime-text#1 to get updates about the color scheme implementation status.

[repo]: https://github.com/arcticicestudio/nord-sublime-text

**Comment by arcticicestudio** [comment]
[3da6d359][c] contains the initial implementation of the color scheme (`Nord.tmTheme`). I'd be happy for some feedback :smile: 

You can test the color scheme by cloning the repository and symlinking it to your packages folder:

```sh
git clone -b feature/gh-1-color-scheme https://github.com/arcticicestudio/nord-sublime-text.git
ln -s nord-sublime-text ~/.config/sublime-text-3/Packages
```

The target package folder path above is for Linux system, make sure to replace the path for macOS / Windows.

[c]: https://github.com/arcticicestudio/nord-sublime-text/commit/3da6d3591f3be054b45674144a6288078208d92a

**Comment by JeffBeltran** [comment]
@arcticicestudio i couldn't get it to load on my setup (i'm on a mac) I did the symlink to `~/Library/Application\ Support/Sublime\ Text\ 3/Packages/` but i can't seem to get the color scheme to load. If i manually set it in the config file it gives me an error and if i search for it under "color scheme..." it doesn't show up, am I missing a step?

**Comment by lee-hyunkyu** [comment]
@JeffBeltran 1. Are you sure you're using Sublime Text 3? 2. If you know you are, use the command palette to change the color scheme. So press Cmd + Shift + P, type in "Change color" and the correct command should be visible. Then look for the color scheme. If you see it, Sublime Text will change the config file correctly. If you don't see it, try just cloning the repository inside the Packages folder rather than just symlinking.

**Comment by arcticicestudio** [comment]
Like @lee-hyunkyu said the theme is build for Sublime Text 3 only. This version supports themes and color schemes natively via the command palette (<kbd>Ctrl</kbd>/<kbd>⌘</kbd>+<kbd>Shift</kbd>+<kbd>p</kbd>). The item *UI: Set Color Scheme* allows to set the color theme while the UI theme can be set with *UI: Select Theme*

**Comment by JeffBeltran** [comment]
yup im on sublime 3 it was an issue with symlinking i guess... i just directly cloned from the packages folder and it showed up, not sure why i didn't just try that.. lol anyways thanks. Looking good

Also where would you like to have the feedback? on this thread? and what about different syntax? should i just list the ones i would like to have working?

**Comment by arcticicestudio** [comment]
You can post feedback in issue arcticicestudio/nord-sublime-text#1.

**Comment by arcticicestudio** [comment]
Closing this issue for now since the theme has already been published some time ago and should work as a basic start. The syntax highlighting theme is available as of [version 0.1.0][v] while the latest state and plan for the UI theme is documented in arcticicestudio/nord-sublime-text#6.

[v]: https://github.com/arcticicestudio/nord-sublime-text/releases/tag/v0.1.0
